livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/models.py

```diff
@@ -26,3 +26,9 @@
     "gpt-3.5-turbo-1106",
     "gpt-3.5-turbo-16k-0613",
 ]
+ImageDetail = Literal["low", "medium", "high"]
+ImageDetailsResizeValues = {
```

Comment:
OAI uses the following in their assistant API: https://platform.openai.com/docs/assistants/how-it-works/low-or-high-fidelity-image-understanding

it seems that our `high` is just matching their `low`? GPT-4o limits things to 768px

Reply:
@keepingitneil when sending an image to OAI and setting the fidelity/detail field, can they resize the image on their end?

Reply:
I believe they can but we still want to resize to avoid sending a large payload. that'll increase latency

Reply:
Yeah so low | medium | high is really only relevant for url images. I re-use it for sizing base64 images but choose different values to better keep the payload size in check. I'll rethink this a bit